textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/regexmatch/GramBooleanQuery.java

```diff
@@ -18,10 +18,10 @@
 		OR
 	}
 	QueryOp operator;
-	List<String> operandList;
-	List<GramBooleanQuery> subQueryList;
+	Set<String> operandSet;
+	Set<GramBooleanQuery> subQuerySet;
 	
-	int gramLength;
+	static int gramLength;
```

Comment:
We make it `static` because `RegexToGramQueryTranslator` needs its value to do translation.

We could have another `gramLength` member in `RegexToGramQueryTranslator`, and pass a value when we initiate a `RegexToGramQueryTranslator` object. Do you think it is a good idea? @zuozhi